### ISSUE TITLE
Better docs to help onboard new contributors

### ISSUE THREAD

**Issue by mkurz** [playframework/play-meta#203]
Probably https://www.playframework.com/documentation/2.8.x/BuildingFromSource isn't doing the job to get people started, we should set up a more detail howto with step by step guides. Also not sure if the referenced page isn't even up to date.
/cc @eloots

**Comment by cleliofs** [comment]
Good point @mkurz I am willing to join the Community, but I am a bit "lost" on how to start and get involved 👍  Ta

**Comment by mkurz** [comment]
@cleliofs I replied to your email you send to the play-team mailing list, thanks!
